extensions/wikia/AdMixExperiment/AdMixExperiment.hooks.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class AdMixExperimentHooks {
+	public static function onBeforePageDisplay() {
+		\Wikia::addAssetsToOutput( 'ad_mix_experiment_js' );
```

Comment:
This way we add additional request for js assets. Can we merge them with original package?